File: Service_eHospital_DongNai_A/Implement/zzzz_SoChungTu_TempService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class zzzz_SoChungTu_TempService : BaseService<zzzz_SoChungTu_Temp, Izzzz_SoChungTu_TempRepository>
    , Izzzz_SoChungTu_TempService
    {
    private readonly Izzzz_SoChungTu_TempRepository _zzzz_SoChungTu_TempRepository;
    public zzzz_SoChungTu_TempService(Izzzz_SoChungTu_TempRepository zzzz_SoChungTu_TempRepository) : base(zzzz_SoChungTu_TempRepository)
    {
    _zzzz_SoChungTu_TempRepository = zzzz_SoChungTu_TempRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/zzzz_SoChungTu_Temp.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { zzzz_SoChungTu_Temp } from './zzzz_SoChungTu_Temp.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class zzzz_SoChungTu_TempService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'SoChungTu', 'SoChungTu_new'];

    List: zzzz_SoChungTu_Temp[] | undefined;
    ListFilter: zzzz_SoChungTu_Temp[] | undefined;
    FormData!: zzzz_SoChungTu_Temp;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "zzzz_SoChungTu_Temp";
    }
}

